### DISCUSSION TITLE
is_empty attribute

### DISCUSSION THREAD

**Discussion by idogrf** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#238] [Q&A] [answered]
Hi,

I was wandering what the is_empty attribute stands for.
I have a first generation xiaomi mi robot and that attribute is always false.
I hoped it would be some indication connected to inserting or pulling out the dust bin from the vacuum but as I said it's always false.

So what is it meant for?

**Comment by PiotrMachowski** [comment, marked as answer]
It tells if map data is present or not

**Reply by idogrf** [reply]
thanks!
